src/query.py

```
@@ -1,48 +1,28 @@
-    """ドキュメントをソース名付きの構造化テキストに整形する。"""
-    chunks = []
-    for i, doc in enumerate(docs, 1):
-        source = os.path.basename(doc.metadata.get("source", "unknown"))
-        chunks.append(f"=== Source {i}: {source} ===\n{doc.page_content}\n---")
-    return "\n\n".join(chunks)
+DOCS_CACHE_PATH = os.path.join(PROJECT_ROOT, "data", "docs_cache.pkl")
 
 
-def query(question: str):
+def query(question: str) -> None:
     """ベクトルストアを使って質問に回答する（CLI 用・履歴なし）。"""
```

Ruff (0.15.7)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)